### PR TITLE
Initial SPM Package

### PR DESCRIPTION
**Pull request by devahmedshendy**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// swift-tools-version: 5.7
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "OpenMaterialUI",
+    platforms: [
+        .iOS(.v16)
```
Comment (0xLeif) [inline Package.swift]: Should we add mac, watch, and tv?

File: Sources/OpenMaterialUI/Text/MaterialText.swift [hunk 2]
```diff
@@ -0,0 +1,40 @@
+//
+//  SwiftUIView.swift
+//  
+//
+//  Created by Ahmed Shendy on 21/05/2023.
+//
+
+import SwiftUI
+
+@available(iOS 16.0, *)
```
Comment (0xLeif) [inline Sources/OpenMaterialUI/Text/MaterialText.swift]: We shouldn't need this if the Package.swift file specifies we are using iOS 16.

File: playground/OpenMaterialUIPlayground/ContentView.swift [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import SwiftUI
+import OpenMaterialUI
+
+struct ContentView: View {
+    var body: some View {
+        VStack {
+            Image(systemName: "globe")
+                .imageScale(.large)
+                .foregroundColor(.accentColor)
+            Text("Hello, world!")
```
Comment (0xLeif) [inline playground/OpenMaterialUIPlayground/ContentView.swift]: Can we change this to use `MaterialText`?

File: playground/OpenMaterialUIPlayground.xcodeproj/project.pbxproj
Comment (0xLeif) [inline playground/OpenMaterialUIPlayground.xcodeproj/project.pbxproj:1]: Nitpick: Can we update the `playground` folder to `Playground`? Normally folders start with a capital letter.

Comment (devahmedshendy) [reply playground/OpenMaterialUIPlayground.xcodeproj/project.pbxproj:1]: @0xLeif  I will have this committed in another PR.

**Review by 0xLeif — APPROVED** [review]

**Pull request merged**